pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -113,6 +120,23 @@ def __call__(self, value, system):
             headers=Config.get('print', {})['headers'],
             data=json.dumps(spec)
         )
+        if Config.get('print', {}).get('compute_toc_pages', False):
+            with io.BytesIO() as pdf:
+                pdf.write(print_result.content)
+                pdf_reader = PdfFileReader(pdf)
+                x = []
+                for i in range(len(pdf_reader.getOutlines())):
+                    x.append(pdf_reader.getOutlines()[i]['/Page']['/StructParents'])
+                true_nb_of_toc = min(x)-1
+
+                if true_nb_of_toc != extract_as_dict['nbTocPages']:
+                    log.warning('nbTocPages in result pdf: {} are not equal to the one guessed : {}, request new pdf'.format(true_nb_of_toc,extract_as_dict['nbTocPages'])) # noqa
+                    extract_as_dict['nbTocPages'] = true_nb_of_toc
+                    print_result = requests.post(
```

Comment:
can we encapsulate that call in a function (which is used by both calls)? I think it would be better for maintainability.

Reply:
I though about that too, but in fact the only function that is called here is `requests.post`.

Reply:
@jwkaltz is [this](https://github.com/openoereb/pyramid_oereb/pull/983/commits/4a42dc7f5fcf2f9728dbb782fc3f89f713024ae9#diff-cfb7a95e0585c750deca6eabb5c5119bR120) ok?